### DISCUSSION TITLE
Can turbsim generate wind speed that changes gradually over time?

### DISCUSSION THREAD

**Discussion by 1335064763fansir** [OpenFAST/openfast#2345] [Q&A]
I want to get a wind speed that increases or decreases over time, what should I do?

**Comment by jjonkman** [comment]
Dear @1335064763fansir,

Normally turbulent wind conditions are simulated about a fixed mean wind speed; i.e., stationarity of the inflow turbulence is assumed. NREL has not provided an engineering tool to calculate the turbulent wind atop a wind ramp. A similar question was discussed on our forum forum--see: https://forums.nrel.gov/t/stitch-together-turbsim-timeseries/2245.

If you are referring to a wind ramp without turbulence, e.g., uniform wind with a ramp for the purposes of generating a power curve within one simulation, these can be generated manually through a uniform wind file.

Best regards,
